parsers/deathloopvaccine.py

```diff
@@ -0,0 +1,286 @@
+import datetime
+import re
+import os
+import signal
+
+import psutil
+
+from datetime import datetime
+from helpers import Parser, config
+
+
+#
+# simple utility to prevent Everquest Death Loop
+#
+# The utility functions by parsing the current (most recent) Everquest log file, and if it detects
+# Death Loop symptoms, it will respond by initiating a system process kill of all "eqgame.exe"
+# processes (there should usually only be one).
+#
+# We will define a death loop as any time a player experiences X deaths in Y seconds, and no player
+# activity during that time.  The values for X and Y are configurable, via the DeathLoopVaccine.ini file.
+#
+# For testing purposes, there is a back door feature, controlled by sending a tell to the following
+# non-existent player:
+#
+#   death_loop:     Simulates a player death.
+#
+#                   Note however that this also sets a flag that disarms the conceptual
+#                   "process-killer gun", which will allow every bit of the code to
+#                   execute and be tested, but will stop short of actually killing any
+#                   process
+#
+#                   The "process-killer gun" will then be armed again after the simulated
+#                   player deaths trigger the simulated process kill, or after any simulated
+#                   player death events "scroll off" the death loop monitoring window.
+#
+class DeathLoopVaccine(Parser):
+
+    """Tracks for DL symptoms"""
+
+    def __init__(self):
+
+        super().__init__()
+        self.name = 'deathloopvaccine'
+
+        # parameters that define a deathloop condition, i.e. D deaths in T seconds,
+        # with no player activity in the interim
+        # todo - make these configarable via the UI?
+        self.deathloop_deaths = config.data['deathloopvaccine']['deaths']
+        self.deathloop_seconds = config.data['deathloopvaccine']['seconds']
```

Comment:
You'll likely want to reference these dynamically from config every time they're used, otherwise config changing won't take effect until a restart.

Reply:
great catch.  fixed in latest push.